packages/config/src/layer2s/metis.ts

```diff
@@ -297,6 +297,13 @@ export const metis: Layer2 = {
       link: 'https://metisdao.medium.com/decentralized-storage-goes-live-da876dc6eb70',
       date: '2022-04-12T00:00:00Z',
       description: 'Update moving data to an off-chain committee.',
+    },    
+    {
```

Comment:
In our understanding this is not a milestone as nothing has changed on the deployed chain. This is about the testnet and so it doesn't qualify. Please remove.

Reply:
sure. I have reverted it now.